tests/devices/unit_tests/test_backlight.py

```diff
@@ -14,3 +16,9 @@ def fake_backlight():
 def test_backlight_can_be_written_and_read_from(fake_backlight: Backlight):
     fake_backlight.pos.sim_put(fake_backlight.IN)
     assert fake_backlight.pos.get() == fake_backlight.IN
+
+
+def test_when_backlight_out_it_switches_off(fake_backlight: Backlight):
```

Comment:
Nit: Could you add the opposite case too? Just so that we cover all paths

Reply:
Done

Reply:
Done